src/NuGet.Core/NuGet.Commands/ListCommand/ListCommandRunner.cs

```diff
@@ -33,7 +33,7 @@ public async Task ExecuteCommand(ListArgs listArgs)
 
             foreach (PackageSource packageSource in listArgs.ListEndpoints)
             {
-                var sourceRepository = Repository.Factory.GetCoreV3(packageSource.Source);
+                var sourceRepository = Repository.Factory.GetCoreV3(packageSource);
```

Comment:
How did you find this? :)

Reply:
Without this change initial PackageSource with CLientCertificates will not be forwarded to HttpHandler :)

Reply:
I imagine you tested this then?

I think this fixes https://github.com/NuGet/Home/issues/2869 which is something I noticed recently when triaging, and I "guessed" that this line is the issue, but didn't get around to test it.

Reply:
Yes this will fix that issue. I've used same deliver mechanism :) Auth information was lost as well

Reply:
Nice, thanks. 
We'll likely have to add some sort of a test to ensure that this is working correctly. 

I'll put a comment in the issue.